src/me/components/Resources.tsx

```diff
@@ -29,83 +30,104 @@ interface Props {
   me: AppState['me']
 }
 
-class ResourceLists extends PureComponent<Props> {
-  public render() {
-    return (
-      <FlexBox
-        direction={FlexDirection.Column}
-        alignItems={AlignItems.Stretch}
-        stretchToFitWidth={true}
-        margin={ComponentSize.Small}
-      >
-        {isFlagEnabled('docSearchWidget') ? (
-          <Panel testID="documentation--panel">
+const ResourceLists: FC<Props> = ({me}) => {
```

Comment:
super weird that this is being prop drilled down here, especially since the parent component isn't making use of the `me` property. If we're already refactoring this, maybe we could just grab that state within the component rather than getting it from the parent?

Reply:
Good point. I checked this out too and great catch, no reason to have this hanging out here.